.travis.yml

```diff
@@ -16,10 +16,8 @@ jobs:
     script:
       - make lint
       - make vendor-check
-  - stage: Test unit and integration
-    script: make test-unit-coverage
-  - stage: Test e2e on private network
-    script: make test-e2e
+  - stage: Test unit, integration and e2e on private network
+    script: make $TEST_SUITE
```

Comment:
I think you can do the same in this way:
```
- stage: Test unit, integration and e2e on private network
  script:
    - make test-unit-race
    - make test-e2e-race  
```

Reply:
It should be exactly the same as in the first example here: https://docs.travis-ci.com/user/build-stages/#How-do-Build-Stages-work%3F

Reply:
What I'm not sure is if we're guaranteed that they will execute on separate machines.

Reply:
I'm not sure I like the idea of stages. For instance, I don't see why we wouldn't run the `Lint & Vendor Check` stage at the same time as the `Test unit, integration and e2e on private network` stage, since they are independent. But not sure what I'd call the stages in that situation.

Reply:
We can get rid of stages. The case was that it does not make sense to run the rest if linter fails. Not sure if Travis stops all concurrent scripts if one fails.